src/System.Private.ServiceModel/tools/setupfiles/BridgeCertificateInstaller.sh

```diff
@@ -3,13 +3,13 @@
 show_banner()
```

Comment:
Nit: consider renaming BridgeCertificateInstaller.sh and BridgeCertificateInstaller.cs.  Not required.